src/main/java/eu/evops/maven/plugins/cucumber/parallel/Run.java

```diff
@@ -141,6 +136,13 @@
     @Parameter(property = "cucumberRunner.combineReports")
     boolean combineReports;
 
+    /**
+     * Controls failing the build if their are test failures. Needed for CI systems such
+     * as Bamboo that allow quarantining of failing tests.
+     */
+    @Parameter(property = "cucumberRunner.test.failure.ignore")
```

Comment:
@chdavi could you rename the property to match the style of the others to ```cucumberRunner.ignoreTestFailure``` please?

Reply:
@sponte Maven will already convert the test.failure.ignore to testFailureIgnore automatically in the xml. I updated the tests to have an example of this. I was trying to keep it as close to the surefire/failsafe property *maven.test.failure.ignore* to make it easy for people to convert their existing scripts and builds.

Do you still want me to change the name?

Reply:
Hey @chdavi, didn't know the conversion happens. I suppose using common/similar name is a good idea. I forgot to ask you to update the README.md so that people are aware of your changes.

I will add CHANGELOG entry when releasing.

Thanks again for contributing.

Reply:
@sponte I updated the README.md. Can you review and make sure its in the format you want. You are welcome. Thanks for making a better solution than the default JUnit setup.